ONSdigital/response-operations-ui
# BUG: Fix error text on verify account page

# What and why?
When a user tries to activate their account and a password fails verification for both length and matching, the incorrect number of errors is displayed at the top of the screen. This needs to be changed to the correct number. This PR fixes the issues.

# How to test?
- Get to the verify account screen (after creating account and clicking link)
- put 1 letter in new password and 1 different letter in the confirm password
- Press 'activate account'

# Trello
https://trello.com/c/fsrNks6A/1861-bug-fix-error-text-on-verify-account-page-track-time
